pkgs/by-name/ld/ld64/0007-Add-OpenSSL-based-CoreCrypto-digest-functions.patch

```diff
@@ -0,0 +1,311 @@
+From 36767c7345161baf0ab125f95c8557f8e24f25db Mon Sep 17 00:00:00 2001
```

Comment:
not trying to be a buzzkill but the compatibility layer here seems like it is lacking tests. how can we tell that the result is the same as without the patch?

looking over this it looks this is just providing implementations of sha1 and sha256 digests?

Reply:
(if builds using ld64 will fail if these digests are broken can ignore my prev comment)

Reply:
ld64 is only using Common Crypto and corecrypto for digests. If it weren’t calculating them correctly, signed binaries would fail to load due to invalid signatures.